app/controllers/candidate_interface/new_references/base_controller.rb

```diff
@@ -0,0 +1,68 @@
+module CandidateInterface
+  module NewReferences
+    class BaseController < CandidateInterfaceController
+      before_action :render_application_feedback_component, :set_reference
+      before_action :redirect_to_dashboard_if_submitted
+
+      def references_review_path
```

Comment:
I think this would be a simple refactor to remove. I don't think these aliasing methods add value and certainly shouldn't be defined in a controller if they are absolutely necessary, perhaps it could be accomplished in the routes file if so?